.github/workflows/release.yml

```
@@ -0,0 +1,195 @@
+          MODL_FILE=$(ls *.modl | head -1)
+          if [ -n "$MODL_FILE" ]; then
+            cp "$MODL_FILE" "Git-${{ steps.version.outputs.version }}-unsigned.modl"
+            echo "Preserved unsigned module: Git-${{ steps.version.outputs.version }}-unsigned.modl"
+            ls -lh "Git-${{ steps.version.outputs.version }}-unsigned.modl"
+          else
+            echo "Error: No .modl file found"
+            exit 1
+          fi
+
+      - name: Decode keystore
+        if: steps.version.outputs.is_release == 'true' && secrets.KEYSTORE_BASE64 != ''
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)